cfgmgr/vxlanmgr.cpp

```diff
@@ -216,13 +216,6 @@ void VxlanMgr::doTask(Consumer &consumer)
 
         if (op == SET_COMMAND)
         {
-            std::string & alias = kfvKey(t);
-            if (!isInterfaceNameLenOk(alias))
-            {
-                it = consumer.m_toSync.erase(it);
-                continue;
-            }
```

Comment:
@arfeigin why is this one was removed?

Reply:
The validation is actually only needed in doVxlanCreateTask (and is done there), validating here will block vxlan tunnels and other entities that doesn't have the restriction of name length.